Makefile

```diff
@@ -5,12 +5,25 @@ OT1_VERSION := 2.5.2
 OT2_VERSION_TAG := v3.3.1-beta.0
 OT2_MONOREPO_DIR := ot2monorepoClone
 
-OT1_FILE_SUFFIX := *.ot1.py
-OT2_FILE_SUFFIX := *.ot2.py
+# Parsers output to here
+BUILD_DIR := protoBuilds
+
+# Ignore all protocol dirs that contain a file named '.ignore'
+# on the top protocol folder level
+IGNORED_INPUT_PATHS := $(addsuffix %, $(dir $(wildcard protocols/*/.ignore)))
+
+OT1_INPUT_FILES_UNFILTERED := $(shell find protocols/ -type f -name '*.ot1.py')
+OT1_INPUT_FILES := $(filter-out $(IGNORED_INPUT_PATHS), $(OT1_INPUT_FILES_UNFILTERED))
+OT1_OUTPUT_FILES := $(patsubst protocols/%.ot1.py, $(BUILD_DIR)/%.ot1.py.json, $(OT1_INPUT_FILES))
+
+OT2_INPUT_FILES_UNFILTERED := $(shell find protocols/ -type f -name '*.ot2.py')
+OT2_INPUT_FILES := $(filter-out $(IGNORED_INPUT_PATHS), $(OT2_INPUT_FILES_UNFILTERED))
+OT2_OUTPUT_FILES := $(patsubst protocols/%.ot2.py, $(BUILD_DIR)/%.ot2.py.json, $(OT2_INPUT_FILES))
 
 .PHONY: install
 install:
```

Comment:
@sfoster1 has pointed out something that I've completely missed. According to the [GNU standard targets doc](https://www.gnu.org/prep/standards/html_node/Standard-Targets.html#Standard-Targets), `make install` is meant to be used to install whatever program the makefile builds, _not_ install build dependencies.

Accordingly, it's probably better to avoid using `install` for these purposes (and I should probably adjust the monorepo's tasks accordingly... ugh). Given the expected remaining lifetime of this repo, it's probably fine to leave this alone if you don't feel like changing it, but yeah, heads up!

Reply:
`make dependencies`???